docs/2_QUICK_START_GUIDE.md

```diff
@@ -1,13 +1,229 @@
+
+* From any terminal with git, use the following git command to clone your forked repo into your local environment. You can find your git repo url at the landing page of your forked repo.
+
+    ```bash
+    git clone <insert-forked-git-repo-url> # ex. git clone https://github.com/<YourGitAccout>/cobalt.git
+    ```
+
+### **Step 3:** Setup Local Environment Variables
+
+You'll need to define a `.env` file in the root of your local project. You can use our [environment template file](https://github.com/microsoft/cobalt/blob/master/.env.template) to start. This will hold all the environment variables needed to run your Cobalt deployments locally.
+
+1. Navigate to the root directory of your project and use the following command to copy the environment template file.
+
+    ```bash
+    cp .env.template .env
+    ```
+
+    ```bash
+    $ tree infra
+    ├───.env.template
+    ├───.env ## NEW FILE GENERATED FROM THE COMMAND
+    ├───modules
+    │   └───providers
+    │       ├───azure
+    │       │   ├───...
+    │       │   └───vnet
+    │       └───common
+    └───templates
+        ├───az-hello-world
+        │   └───test
+        └───backend-state-setup
+    ```
+
+1. Provide values for the environment values in `.env` which are required to authenticate Terraform to provision resources within your subscription.
+
+    ```bash
+    # Versioning
+    GO_VERSION=1.12.5
+    TF_VERSION=0.12.2
+    BUILD_BUILDID=1
+    # Subscription Values
+    ARM_SUBSCRIPTION_ID="<az-service-principal-subscription-id>"
+    ARM_CLIENT_ID="<az-service-principal-client-id>"
+    ARM_CLIENT_SECRET="<az-service-principal-auth-secret>"
+    ARM_TENANT_ID="<az-service-principal-tenant>"
+    ARM_ACCESS_KEY="<remote-state-storage-account-primary-key>"
+    # Azure Blob Storage Values
+    TF_VAR_remote_state_account="<tf-remote-state-storage-account-name>"
+    TF_VAR_remote_state_container="<tf-remote-state-storage-container-name>"
+    TF_WARN_OUTPUT_ERRORS=1
+    TF_CLI_ARGS=
+    ```
+
+1. Execute the following commands to set up the environment variables for your bash session:
+
+    ```bash
+    # these commands setup all the environment variables needed to run this template
+    DOT_ENV=<path to your .env file>
+    export $(cat $DOT_ENV | xargs)
+    ```
+
+1. Execute the following login command to configure your local Azure CLI.
+
+    ```bash
+    # This logs your local Azure CLI in using the configured service principal.
+    az login --service-principal -u $ARM_CLIENT_ID -p $ARM_CLIENT_SECRET --tenant $ARM_TENANT_ID
+    ```
+
+    ```bash
+    # Other helpful azure cli commands
+    az --version # Cobalt does not recommend a version but this command proves useful for troubleshooting the az cli
+    az account show # A command for validating the azure cli is correctly configured
+    az configure --defaults group='' # Use empty strings to clear default values for any sticky az properties like "group"
```

Comment:
Is this necessary now that all the cmds use a specific RG and Sub?

Reply:
That's basically an opinion about an azure cli pitfall I ran into. Helpful but maybe not the right place for it. I'll remove it for now. Thanks.

Reply:
I left "az account show" but removed the others.